src/datreant/metadata.py

```diff
@@ -452,7 +452,8 @@ def add(self, categorydict=None, **categories):
                 if not isinstance(key, string_types):
                     raise TypeError("Keys must be strings.")
 
-                if (isinstance(value, (int, float, string_types, bool))):
+                if (isinstance(value, (int, float, string_types,
```

Comment:
This will have to be documented, so examples of assigning a list of values to a category